### PR TITLE
Justinbarry/ch8749/add docker compose yml to support ui dev

### PR DESCRIPTION
**Pull request by justinbarry**
[Clubhouse Story](https://app.clubhouse.io/augur/story/8749)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+version: "3.2"
+services:
+  geth:
+    image: augurproject/dev-pop-geth:latest
+    ports:
```
Comment (bthaile) [inline docker-compose.yml]: we don't want augurproject/dev-pop-geth:latest

we want the docker version that matches the augur-core version that augur.js uses. 

I think we need to call >npm explore augur.js -- npm run core:version to get the core-#### version of pop docker.

**Comment by bthaile** [comment]
very close just need one configuration details.

File: docker-compose.yml (hunk 1 above)
Comment (justinbarry) [reply docker-compose.yml]: @bthaile Getting this error:
```
✗ npm explore augur.js -- npm run core:version
npm ERR! missing script: core:version

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/justin/.npm/_logs/2018-04-09T12_49_14_561Z-debug.log
npm ERR! weird error 1
```

Comment (justinbarry) [reply docker-compose.yml]:
```
npm explore augur.js -- npm run  --silent core:version
```
The added --silent suppresses the output.

**Comment by justinbarry** [comment]
@bthaile Can you move this where you see appropriate?
To start:
`AUGUR_CORE_VERSION=$(npm explore augur.js -- npm run  --silent core:version) docker-compose up`
To kill all the containers:
`AUGUR_CORE_VERSION=$(npm explore augur.js -- npm run  --silent core:version) docker-compose down`

**Comment by justinbarry** [comment]
What you mean?

**Comment by bthaile** [comment]
created driver/caller by way of npm command.

**Pull request merged**